lib/base/model.js

```diff
@@ -584,12 +592,18 @@ ModelBase.prototype.saveMethod = function(options) {
 
   if (options.patch) {
     if (options.method === 'insert')
-      throw new TypeError(`Cannot accept incompatible options: method=insert, patch=${options.patch}`);
+      throw new TypeError(
+        `Cannot accept incompatible options: method=insert, patch=${
+          options.patch
+        }`
+      );
 
     options.method = 'update';
   }
-  return (options.patch && 'update' || options.method) == null
-    ? (this.isNew() ? 'insert' : 'update')
+  return ((options.patch && 'update') || options.method) == null
+    ? this.isNew()
+      ? 'insert'
+      : 'update'
```

Comment:
We should burn this evil nested ternary operator, but that has nothing to do with this PR. This is more of a note for my future self.